src/types/domain.ts

```diff
@@ -7,6 +7,7 @@ export type Event = {
   location: string;
   imageUrl: string;
   imageAlt: string;
+  eventType: string;
```

Comment:
`eventType` should be a discriminated union of string literals to give better auto-completion and type-checking. E.g.
```ts
eventType: 'Subject matter event' | 'Kodekveld' | ... // etc.
```